src/components/chart/_macro.spec.js

```diff
@@ -240,6 +243,36 @@ describe('Macro: Chart', () => {
                 });
             });
         });
+
+        describe('GIVEN: Params: Legend is set to true', () => {
+            describe('WHEN: legend parameter is provided and set to true', () => {
+                const $ = cheerio.load(renderComponent('chart', EXAMPLE_LINE_CHART_WITH_LEGEND_TRUE_PARAMS));
+                test('THEN: it renders the legend', () => {
+                    const configScript = $(`script[data-highcharts-config--line-chart-legend-true-123]`).html();
+                    expect(configScript).toContain('"legend":{"enabled":true}');
+                });
+            });
+        });
+
+        describe('GIVEN: Params: Legend is set to false', () => {
+            describe('WHEN: legend parameter is provided and set to false', () => {
+                const $ = cheerio.load(renderComponent('chart', EXAMPLE_LINE_CHART_WITH_LEGEND_FALSE_PARAMS));
+                test('THEN: it does not render the legend', () => {
+                    const configScript = $(`script[data-highcharts-config--line-chart-legend-false-123]`).html();
+                    expect(configScript).toContain('"legend":{"enabled":false}');
+                });
+            });
+        });
+
+        describe('GIVEN: Params: Legend is not provided', () => {
```

Comment:
```suggestion
        describe('GIVEN: Params: Legend', () => {
```

Reply:
Query: in this case we wouldn't be passing a legend parameter - is there a better 'given' we could use here?

Reply:
(I have updated as suggested for now)

Reply:
I think this test is concerning the legend param though and thats the style we have adopted when writing our tests